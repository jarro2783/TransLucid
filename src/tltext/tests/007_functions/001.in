fun foo.a.b [a imp intmp, b imp intmp] = if a > b then a else b fi;;
fun foo.a.b [a imp ustring, b imp ustring] = "strings";;
%%
foo.1.2;;
foo.2.1;;
foo.2.4;;
foo."hello"."world";;
