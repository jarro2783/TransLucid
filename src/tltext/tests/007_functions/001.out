2
2
4
"strings"
